conans/client/tools/oss.py

```diff
@@ -33,6 +33,16 @@ def cpu_count(output=None):
     return 1  # Safe guess
 
 
+def detected_os():
+    if OSInfo().is_macos:
+        return "Macos"
+    if OSInfo().is_windows:
+        return "Windows"
+    if OSInfo().is_solaris:
```

Comment:
Not sure about this. The declared OS in the default ``settings.yml`` is ``SunOS``, not ``Solaris``